io.jor.nanosaur2.yml

```diff
@@ -0,0 +1,37 @@
+app-id: io.jor.nanosaur2
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: Nanosaur2
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: nanosaur2
+    buildsystem: cmake
+    sources:
```

Comment:
sources are usually the last in a module.

Reply:
OK, will move it

Reply:
Done in [684d9c0](https://github.com/flathub/flathub/pull/3885/commits/684d9c02a13e3a9cfaea9d9f8fce8096ba1a5190)